docs/develop/development-kit-overview.md

```diff
@@ -1,21 +1,17 @@
 ---
-title: Create Office Add-in projects using Office Add-ins Development Kit for Visual Studio Code (preview)
+title: Create Office Add-in projects using Office Add-ins Development Kit for Visual Studio Code 
 description: Learn how to create Office Add-in projects using Office Add-ins Development Kit extension.
-ms.date: 09/09/2024
+ms.date: 11/12/2024
```

Comment:
the ms.date value has to be American style, with the month first and then the day of the month. Also, be sure to update the ms.date on the two quickstart articles and the article about setting up the developer environment.

```suggestion
ms.date: 12/11/2024
```

Reply:
Done